src/app/bug/shared/bug.service.spec.ts

```diff
@@ -1,11 +1,17 @@
 import { HttpClient } from '@angular/common/http';
 import { HttpClientTestingModule } from '@angular/common/http/testing';
 import {  SpectatorHttp, createHttpFactory } from '@ngneat/spectator'
+import { BugDataService } from './bug-data.service';
 import { BugService } from './bug.service';
+import * as faker from 'faker';
+import { Month } from './models/month.model';
+import { Bug } from './models/bug.model';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/JohnnyGitH/BugEDex/pullRequest?prid=8845963)
```suggestion
import { Bug } from "./models/bug.model";
```